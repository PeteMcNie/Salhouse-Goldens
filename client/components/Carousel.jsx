import React, { useState } from 'react'

import Carousel from 'react-bootstrap/Carousel'

function ControlledCarousel () {
  const [index, setIndex] = useState(0)

  const handleSelect = (selectedIndex, e) => {
    setIndex(selectedIndex)
  }

  return (
    <Carousel activeIndex={index} onSelect={handleSelect}>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/09.jpg"
          alt="First slide"
        />
        <Carousel.Caption>
          <h3>First slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/04.jpg"
          alt="Second slide"
        />

        <Carousel.Caption>
          <h3>Second slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/12.jpg"
          alt="Third slide"
        />

        <Carousel.Caption>
          <h3>Third slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/11.jpg"
          alt="Third slide"
        />

        {/* <Carousel.Caption>
          <h3>Third slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption> */}
      </Carousel.Item>
    </Carousel>
  )
}

export default ControlledCarousel
