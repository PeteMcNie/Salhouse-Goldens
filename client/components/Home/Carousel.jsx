import React, { useState } from 'react'

import Carousel from 'react-bootstrap/Carousel'

function ControlledCarousel () {
  const [index, setIndex] = useState(0)

  const handleSelect = (selectedIndex, e) => {
    setIndex(selectedIndex)
  }

  return (
    <Carousel activeIndex={index} onSelect={handleSelect}>
      <Carousel.Item>
        <img
          className="d-block w-100;"
          src="images/lassie.jpg"
          alt="First slide"
        />
        <Carousel.Caption>
          <h3>First slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/maddie.jpg"
          alt="Second slide"
        />

        <Carousel.Caption>
          <h3>Second slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/munch.jpg"
          alt="Third slide"
        />

        <Carousel.Caption>
          <h3>Third slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          className="d-block w-100"
          src="images/11.jpg"
          alt="Forth slide"
        />

        {/* <Carousel.Caption>
          <h3>Third slide label, do you want this?</h3>
          <p>Do you want to add further comments below the label?</p>
        </Carousel.Caption> */}
      </Carousel.Item>
    </Carousel>
  )
}

export default ControlledCarousel
