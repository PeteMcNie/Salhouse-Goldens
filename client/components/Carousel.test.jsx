import React from 'react'
import { render } from '@testing-library/react'

import Carousel from './Carousel'

test('Carousel component to match snapshot', () => {
  const { container } = render(<Carousel />)
  expect(container).toMatchSnapshot()
})
