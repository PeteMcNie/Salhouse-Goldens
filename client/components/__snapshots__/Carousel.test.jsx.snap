// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel component to match snapshot 1`] = `
<div>
  <div
    class="carousel slide"
  >
    <ol
      class="carousel-indicators"
    >
      <li
        class="active"
      />
      <li />
      <li />
      <li />
    </ol>
    <div
      class="carousel-inner"
    >
      <div
        class="active carousel-item"
      >
        <img
          alt="First slide"
          class="d-block w-100"
          src="images/09.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            First slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Second slide"
          class="d-block w-100"
          src="images/04.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            Second slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Third slide"
          class="d-block w-100"
          src="images/12.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            Third slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Forth slide"
          class="d-block w-100"
          src="images/11.jpg"
        />
      </div>
    </div>
    <a
      class="carousel-control-prev"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-prev-icon"
      />
      <span
        class="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      class="carousel-control-next"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-next-icon"
      />
      <span
        class="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
